spec/system/planning_applications/reviewing/sign_off_spec.rb

```diff
@@ -60,9 +69,8 @@
 
     expect(list_item("Sign-off recommendation")).to have_content("Complete")
 
-    click_link "Back"
+    click_on "Publish decision"
```

Comment:
Changing the route to get to the publish determination page but everything else stays the same.